src/components/diff-view.jsx

```diff
@@ -42,64 +42,53 @@ export default class DiffView extends React.Component {
    */
   componentWillReceiveProps (nextProps) {
     if (this._canFetch(nextProps) && !this._propsSpecifySameDiff(nextProps)) {
-      this._loadDiff(nextProps.page.uuid, nextProps.a.uuid, nextProps.b.uuid, nextProps.diffType);
+      this._loadDiffData(nextProps.page.uuid, nextProps.a.uuid, nextProps.b.uuid, nextProps.diffType);
     }
   }
 
   render () {
-    const {diffType} = this.props;
-    const {diff} = this.state;
-
-    if (diffType && diffTypes[diffType].diffService === 'TODO') {
```

Comment:
I got rid of this check because we don't have any 'TODO' services anymore which we were using to track the diff types that we wanted for -ui. In the future, we would probably just add any additional diff service when needed and not mark it as TODO.

Reply:
Makes sense to me. :+1: